dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.10.5-bullseye
+
+RUN apt-get update && apt-get install
+
+WORKDIR /log_app
+
+COPY data_io.py main.py operations.py requirements.txt ./
```

Comment:
btw it would be better to keep application code in separate directory like /app and the rest that is miscellaneous in root like dockerfile, requirements.txt. It is good practise.

Reply:
You're right, I will refactor it in a second. Hope it won't break imports etc. :)